### PR TITLE
Add elements to rec info for wispi api level and NCP app version

### PR DESCRIPTION
**Pull request by tkojak357**
Adding elements to Rec Info for the ESP's Wispi API Level and the ESP's application FW version number. Tried to keep it general with "Network Co-Processor", but I'm very open to name suggestions.

### CODE REVIEW

**Comment by pscheidler** [comment]
This looks good to me, but definitely hold off on merging until David reviews

**Review by pscheidler — COMMENTED** [review]
Looks good, but hold off on merging until David approves

**Review by StokesMIDE — APPROVED** [review]

**Pull request merged**
